bundles/org.eclipse.ui.ide/src/org/eclipse/ui/internal/ide/messages.properties

```diff
@@ -1049,7 +1049,7 @@ ChooseWorkspaceDialog_recentWorkspaces=&Recent Workspaces
 ChooseWorkspaceDialog_ResolvedAbsolutePath=Full path: {0} 
 ChooseWorkspaceDialog_TildeNonExpandedWarning=\u26A0\uFE0F '~' is not expanded, full path: {0}
 ChooseWorkspaceDialog_InvalidPathWarning=\u26A0\uFE0F The path is invalid on this system: {0}
-ChooseWorkspaceDialog_NotWriteablePathWarning=\u26A0\uFE0F The path is not writable by the current user: {0} 
+ChooseWorkspaceDialog_NotWriteablePathWarning=\u26A0\uFE0F The path may not writable by the current user: {0} 
```

Comment:
`The path may not **be** writable by the current user: {0}`

Reply:
thanks